java/src/test/java/com/saucelabs/simplesauce/SauceSessionTest.java

```diff
@@ -122,14 +109,52 @@ public void sauceOptions_startWithChrome_startsChrome() {
 
     @Test(expected = NullPointerException.class)
     public void stop_newWebDriverInstanceSetByStart_stopsSession() {
-        sauce = new SauceSession(dummyRemoteDriver, dummyEnvironmentManager);
+        sauce = new SauceSession(dummyRemoteDriver);
 
         WebDriver driver = sauce.start();
         sauce.stop();
 
         driver.quit();
     }
 
+    @Test
+    public void usesENVAsDefaultUsername() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4732268)